magda-web-server/package.json

```diff
@@ -10,19 +10,36 @@
     "start": "node dist/index.js",
     "dev": "run-typescript-in-nodemon src/index.ts",
     "docker-build-local": "create-docker-context-for-node-component --build --push --tag auto --local",
-    "docker-build-prod": "create-docker-context-for-node-component --build --push --tag auto"
+    "docker-build-prod": "create-docker-context-for-node-component --build --push --tag auto",
+    "test": "mocha --compilers ts:@magda/scripts/node_modules/ts-node/register --require @magda/scripts/node_modules/tsconfig-paths/register \"src/test/**/*.spec.ts\""
   },
   "devDependencies": {
     "@magda/scripts": "^0.0.30-0",
+    "@types/chai": "^4.0.1",
     "@types/config": "0.0.32",
-    "@types/express": "^4.0.35",
+    "@types/express": "^4.0.39",
+    "@types/lodash": "^4.14.68",
+    "@types/mocha": "^2.2.41",
+    "@types/nock": "^8.2.1",
+    "@types/sinon": "^2.3.3",
+    "@types/supertest": "^2.0.2",
     "@types/urijs": "^1.15.34",
+    "@types/xml2js": "^0.4.0",
     "@types/yargs": "^6.6.0",
-    "typescript": "~2.5.0"
+    "chai": "^4.1.0",
+    "mocha": "^3.4.2",
+    "nock": "^9.0.14",
+    "sinon": "^2.4.1",
+    "supertest": "^3.0.0",
+    "typed-promisify": "^0.4.0",
+    "typescript": "~2.5.0",
+    "xml2js": "^0.4.19"
   },
   "dependencies": {
     "@magda/web-client": "^0.0.30-0",
+    "@magda/typescript-common": "^0.0.30-0",
     "express": "^4.15.3",
+    "sitemap": "^1.13.0",
```

Comment:
This should be a devDependency.  It's needed a build time, but not at runtime.

Reply:
Are you sure? The sitemap generation is all done on-the-fly, not as a build thing, right... or did I mess it up? 😬

Reply:
Oops sorry you're right.  For some reason I thought I was looking at client-side code when I saw the bit that required-in sitemap.  But it's node.js code, so yeah it needs to be in dependencies.